enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -266,6 +200,34 @@ const std::set<std::string> KnownInactiveFunctions = {
     "logbl",
 };
 
+const char *DemangledKnownInactiveFunctionsStartingWith[] = {
+    "f90io",
```

Comment:
Can this be a separate category. In particular the fortran f90io as an example doesn't belong here.

Reply:
afaik only ss5print and f90io are non-c++ names? So I moved them back to check unconditionally for those unmangled.